graphql/query.go

```diff
@@ -78,15 +79,11 @@ var listArgs = graphql.FieldConfigArgument{
 	},
 }
 
-func listParamResolver(r *resource.Resource, p graphql.ResolveParams, params url.Values) (lookup *resource.Lookup, offset int, limit int, err error) {
+func listParamResolver(r *resource.Resource, p graphql.ResolveParams, params url.Values) (q *query.Query, err error) {
```

Comment:
FYI: For _later_ improvement only.

As per previous discussion on `New()`, I think the goal of a New function should be to reduce the code, error checking and validation necessary in this an similar functions. I don't know if it would help, but here is an example from the top of my head:

`query` package:
```go
// Parser should extract the raw string values and
// return them for further validation by e.g. New.
type Parser interface{
        Projection() string
        Filter() string
        Sort() string
        Window() *Window
}

// Parse returns a new Query based on the raw values extracted from p. 
Parse(p Parser) (*Query, error) {...}
```
In this package (and similar in the `rest` package):
````go
type QueryParser struct{}

NewQueryParser(p graphql.ResolveParams, params url.Values) (*QueryParser, error)
```